contracts/solidity/NFTXVaultUpgradeable.sol

```diff
@@ -82,6 +81,9 @@ contract NFTXVaultUpgradeable is
         is1155 = _is1155;
         allowAllItems = _allowAllItems;
         emit VaultInit(vaultId, _assetAddress, _is1155, _allowAllItems);
+        
+        // set manager to the caller so that we can call `setVaultFeatures`
+        manager = msg.sender;
```

Comment:
Is this not immediately overwritten by `contracts/solidity/NFTXVaultFactoryUpgradeable.sol` on line 303, as we call:
```
NFTXVaultUpgradeable(newBeaconProxy).setManager(msg.sender);
```
straight after the `__NFTXVault_init` call?